recipe/meta.yaml

```diff
@@ -24,14 +24,14 @@ requirements:
   host:
     - python
     - pip
-    - numpy >=1.13.3
+    - numpy >=1.16.0
     - cython
 
   run:
     - python
     - scipy >=1.0.0
-    - numpy >1.13.3
-    - matplotlib-base >=1.5.1,<3.4
+    - numpy >1.16.0
+    - matplotlib-base >=1.5.1
```

Comment:
Removed the max value here, technically 3.4.0 is no good for py3.6, but we didn't see any issues with this in CI or locally when testing the testpypi release, so I assume that matplotlib has fixed it's limits for python versions now.